.travis.yml

```diff
@@ -1,7 +1,17 @@
 sudo: false
 language: python
+env:
+  - MOZ_HEADLESS=1
+
+addons:
+  firefox: "55.0"
+
 python:
   - "3.6"
+before_install:
+  - wget https://github.com/mozilla/geckodriver/releases/download/v0.20.1/geckodriver-v0.20.1-linux64.tar.gz
+  - tar -xzvf geckodriver-v0.20.1-linux64.tar.gz
+  - sudo mv geckodriver /usr/local/bin
 install:
   - "pip install -r requirements/dev.txt"
```

Comment:
AFAIK, its not required but am not sure... Clearly that part of the file was not added by me the document structure was maintained... I believe if I have to modify anything other than the things I added this should be via a separate issue and PR.

Reply:
Let us make the code consistent where we can possibly do. In future as well, if something seems inconsistent and could be easily changed, please go ahead and make the changes along with your current work. Yeah I agree separate PR makes sense, but since you are touching this code, minor modifications for overall quality and consistencies can be done.